File: MarvelousMashupServerTeam23/src/main/java/communication/messages/events/game/PauseStartEvent.java
package communication.messages.events.game;


import communication.messages.Message;
import communication.messages.enums.EventType;

/**
 * This message is sent to all clients to inform them about the beginning of a pause
 *
 * @author Sarah Engele
 *
 */
public class PauseStartEvent extends Message {


    /**
     * the constructor of the PauseStartEvent-Class. Creates a PauseStartEvent-MessageObject.
     *
     * @author Sarah Engele
     */
    public PauseStartEvent(){
       super(EventType.PauseStartEvent);
    }
}

File: MarvelousMashupTeam16/Assets/Scripts/DataClasses/Comms/events/game/PauseStartEvent.cs
/**
 * This message is sent to all clients to inform them about the beginning of a pause
 *
 * @author Sarah Engele
 *
 */
public class PauseStartEvent : Message, GameEvent
{
    /**
     * the constructor of the PauseStartEvent-Class. Creates a PauseStartEvent-MessageObject.
     *
     * @author Sarah Engele
     */
    public PauseStartEvent() : base(EventType.PauseStartEvent)
    {
       
    }
    
    public void Execute()
    {
        // TODO: No need for this yet!
    }
}
